schemes/bfv/bfv.go

```diff
@@ -1,180 +1,4 @@
 // Package bfv provides an RNS-accelerated implementation of the Fan-Vercauteren version of Brakerski's (BFV) scale-invariant homomorphic encryption scheme.
-// The BFV scheme enables SIMD modular arithmetic over encrypted vectors or integers.
+// The BFV scheme enables SIMD modular arithmetic over encrypted vectors or integers. See `README.md` for more information
+// on how to instantiate a BFV evaluator.
 package bfv
```

Comment:
Is there a good reason to keep an empty & deprecated package ?

I suggest to either remove the `bfv.go` file so that the folder is no longer a Go package, OR to keep the `bfv` API for v6 with a [deprecation notice](https://go.dev/wiki/Deprecated).

Reply:
I removed the `bfv.go` file. The README is clear enough.